packages/core-modules/test/contracts/modules/FeatureFlagModule.test.js

```diff
@@ -0,0 +1,137 @@
+const { ethers } = hre;
```

Comment:
in general this file is not following the same testing approach we have settled upon in the `synthetix-main`. In general, tests should be focused on breakign down into each writable function, testing all the different permutations of arguments you might submit to the function, verifying that the corresponding read functions are behaving as expected, etc. This tends to result in very high and quality test coveage.

Please take a look at one of the `synthetix-main` test files, such as `VaultModule` for example. Then can you come back and restructure these tests a bit to have a similar structure?

Reply:
yea these can be organized a bit better.   all paths are tested but are a bit hard to follow, i'll reorg in a separate PR and have it look similar to VaultModule, thanks for the link